AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -3648,3 +3648,4 @@ post-gazette.com#$##toolbar > #share-bar { display: none!important; }
 ||kfund-media.com/wp-content/uploads/2018/01/580h400_banner_telegram-min.png
 ||mayu-pravo.com/share/slider/
 ||u.today/img/satoshi-twitter.png
+tinhte.vn##.snd.jsx-3208106821
```

Comment:
Too wide rule, it blocks a lot of things. I'll fix it here to show you what we do in such cases.